docs/cep/tutorials/executing-scripts.md

```diff
@@ -4,9 +4,9 @@ sidebar_position: 10
 
 # Executing Scripts
```

Comment:
Please delete this line and put the title in the metadata above. 
```
---
title: Executing Scripts
sidebar_position: 10
---
```

Reply:
Done